File: Algo/hr_strings/findcamelcaseWords.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

int camelcase(string s)
{
    int count = 1;

    for (int i = 0; i < s.size(); i++)
    {
        int asciiVal = (int)s[i];
        if (asciiVal >= 65 && asciiVal <= 90)
        {
            count++;
        }
    }

    return count;
}

int main()
{
    cout << "Ans: " << camelcase("saveChangesInTheEditor") << endl;
    return 0;
}

File: Algo/hr_strings/findcamelcaseWords.js
function camelcase(s) {
  let count = 1;
  for (let w in s) {
    let asciiVal = s[w].charCodeAt(0);
    if (asciiVal >= 65 && asciiVal <= 90) {
      count++;
    }
  }
  return count;
}
console.log('Ans: ', camelcase('saveChangesInTheEditor'));
